<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCameraViewfinder" name="viewFinder" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Viewfinder Screen</string>
    </property>
    <property name="whatsThis">
     <string>Viewfinder Screen</string>
    </property>
    <property name="accessibleName">
     <string>Viewfinder Screen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonGetShapshot">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSnapshot">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>341</width>
      <height>331</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelExposureCompensation">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>311</y>
      <width>120</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Exposure Compensation:</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderExposureCompensation">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>120</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-4</number>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="pageStep">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionToggleCamera"/>
    <addaction name="actionToggleLock"/>
   </widget>
   <widget class="QMenu" name="menuDevices">
    <property name="title">
     <string>Devices</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDevices"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res/res.qrc">
     <normaloff>:/images/power.png</normaloff>:/images/power.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionToggleCamera">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res/res.qrc">
     <normaloff>:/images/camera.png</normaloff>:/images/camera.png</iconset>
   </property>
   <property name="text">
    <string>Toggle Camera</string>
   </property>
   <property name="statusTip">
    <string>Toggle Camera</string>
   </property>
   <property name="whatsThis">
    <string>Toggle Camera</string>
   </property>
  </action>
  <action name="actionToggleLock">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res/res.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>Toggle Lock</string>
   </property>
   <property name="iconText">
    <string>Toggle Lock</string>
   </property>
   <property name="toolTip">
    <string>Toggle Lock</string>
   </property>
   <property name="statusTip">
    <string>Toggle Lock</string>
   </property>
   <property name="whatsThis">
    <string>Toggle Lock</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCameraViewfinder</class>
   <extends>QWidget</extends>
   <header>qcameraviewfinder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections/>
</ui>
